inferno/io/transform/image.py

```diff
@@ -664,14 +664,15 @@ def build_random_variables(self):
 
     def batch_function(self, image):
         scale = self.get_random_variable('seg_scale')
-        image_shape = np.array(image[0].shape[1:])
-
+        input_image, segmentation = image
+        image_shape = np.array(input_image.shape[1:])
+        if input_image.ndim == segmentation.ndim + 1:
```

Comment:
As far as I understand this change is necessary if you have a segmentation without channel dimension.
Which implies that `image` will be a list of tensors and not a single tensor.
Rather than making this change, could you add a channel dimension to `segmentation` earlier and then give
`image` as a single tensor containing the image and the segmentation?